pkg/hubble/parser/parser.go

```diff
@@ -26,6 +28,8 @@ import (
 	monitorAPI "github.com/cilium/cilium/pkg/monitor/api"
 	"github.com/cilium/cilium/pkg/proxy/accesslog"
 	"github.com/golang/protobuf/ptypes"
+	"google.golang.org/protobuf/types/known/timestamppb"
+	"google.golang.org/protobuf/types/known/wrapperspb"
 
 	"github.com/golang/protobuf/ptypes/wrappers"
```

Comment:
See above, this one here imports the same types as `"google.golang.org/protobuf/types/known/wrapperspb"`

Reply:
Thanks. Will drop `google.golang.org/protobuf/types/known/wrapperspb`.